README.md

```diff
@@ -180,6 +180,142 @@ steps:
   - run: go run hello.go
   ```
 
+**Caching in multirepos**
+`cache-dependency-path` input assepts glob patterns and multi-line values:
+
+```yaml
+steps:
+  - uses: actions/checkout@v3
+  - uses: actions/setup-go@v4
+    with:
+      go-version: '1.17'
+      cache-dependency-path: **/go.sum
+  - run: go run hello.go
+  ```
+
+```yaml
+steps:
+  - uses: actions/checkout@v3
+  - uses: actions/setup-go@v4
+    with:
+      go-version: '1.17'
+      cache-dependency-path: |
+        subdir1/go.sum
+        subdir2/go.mod
+  - run: go run hello.go
+  ```
+
+## Multi-target builds
+`cache-dependency-path` input used to generate unuque cache key and it is not limited to only
+dependencies files. The common case is to add a file containing the extr info about the specific
+build, for example build target.
+
+```yaml
+env:
+  GOOS: ...
+  GOARCH: ...
+
+steps:
+  - run: echo "$GOOS $GOARCH"> /tmp/env
```

Comment:
```suggestion
  - run: echo "$GOOS $GOARCH" > /tmp/env
```
Not sure about the whitespace between the string and >. If It breaks the command, just ignore the comment.